docs/two-phase-commit-transactions.md

```diff
@@ -605,28 +597,28 @@ public class Sample {
             .filter(Objects::nonNull)
             .collect(Collectors.toList());
 
-    // If any of the transactions succeeded to commit, you can regard the transaction as committed
+    // If any of the transactions successfully committed, you can regard the transaction as committed.
     if (exceptions.size() < transactions.length) {
       if (!exceptions.isEmpty()) {
-        // You can log the exceptions here if you want
+        // You can log the exceptions here if you want.
       }
 
-      return; // Succeeded to commit
+      return; // Commit was successful.
     }
 
     //
     // If all the transactions failed to commit:
     //
```

Comment:
This comment seems to slightly contradict the next comment, which mentions "If **any** of the transactions failed to commit ...."

Is there content missing here?

Reply:
@brfrn169 Thank you for the explanation. I think I understand this part now.

How about we reword it as follows?

```suggestion
    //
    // If all the transactions failed to commit, one of the following exceptions will be thrown as described:
    //
```

Reply:
Maybe this change is not necessary.

This logic is that, we basically throw the first exception listed in `exceptions`, but if any of the `exceptions` is `UnknownTransactionStatusException`, we prioritize throwing it instead. So I think we can keep it as it is. Thank you.

Reply:
OK, I'll keep this as is for now since it's safe to assume that users will understand this part.